core/libraries/batch/JobHandlers/RegistrationsReport.php

```diff
@@ -291,6 +297,36 @@ public function get_csv_data_for($event_id, $offset, $limit, $question_labels, $
                     $reg_row['Event_CPT.ID']
                 );
             }
+            // add attendee columns
+            foreach ($att_fields_to_include as $att_field_name) {
+                $field_obj = EEM_Attendee::instance()->field_settings_for($att_field_name);
+                if ($reg_row['Attendee_CPT.ID']) {
+                    if ($att_field_name == 'STA_ID') {
+                        $value = EEM_State::instance()->get_var(
```

Comment:
good stuff, looks like you are having no troubles adapting to use the model system 👍🏻